### PR TITLE
Add support for prepending url to scripts output url

### PR DESCRIPTION
**Pull request by mrexclamation**
I found a scenario where you need to prepend url to output assets url without touching the baseHref on document.

Output example :

**`<link href="src.4b3697e6ebd39ce5cf64.css" rel="stylesheet">`**

With **prependUrl** option set to **'/assets/'** :

**`<link href="/assets/src.4b3697e6ebd39ce5cf64.css" rel="stylesheet">`**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,18 +25,19 @@ There are a couple required parameters:
 
 And some other optional:
 
-- `appMountId`: div element id on which you plan to mount a javascript app (can include multiple elements using the `appMountIds` array)
-- `devServer`: insert the webpack-dev-server hot reload script at this host:port/path (eg, http://localhost:3000)
-- `baseHref`: Adjust the url for relative urls in the document ([MDN](https://developer.mozilla.org/en/docs/Web/HTML/Element/base))
+- `appMountId`: div element id on which you plan to mount a javascript app (can include multiple elements using the `appMountIds` array).
```
Comment (jaketrent) [inline README.md:28]: Awesome, thanks for the consistency adjustments.

Comment (mrexclamation) [reply README.md:28]: Yep I like that !

File: index.ejs [hunk 2]
```diff
@@ -66,7 +67,7 @@
 <% } %>
 
 <% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
-<script src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
+<script src="<%=prependUrl%><%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
```
Comment (jaketrent) [inline index.ejs:70]: I don't quite understand the use case of why this is useful.  These chunks and their paths are all handled by webpack.  This seems liked something that `publicPath` could solve.  https://github.com/webpack/docs/wiki/Configuration#outputpublicpath ...  Have a look, please, and let me know what you think.

File: README.md (hunk 1 above)
Comment (mrexclamation) [reply README.md:28]: Its a good idea to have the implementation I made ?

File: index.ejs (hunk 2 above)
Comment (mrexclamation) [reply index.ejs:70]: DOH ! I didn't remember that one !

I'll make a test (which I'm sure will work) and I let you know ...

Thanks and apologize me !

Comment (jaketrent) [reply index.ejs:70]: No problem.  Good luck on your test!

Comment (jaketrent) [reply index.ejs:70]: Just another note...  If you'd like to submit another PR or adjust this one to only include the README changes, I'll merge that one.

Comment (mrexclamation) [reply index.ejs:70]: Man thank you so much, maybe I was in such a hurry that I fogot publicPath :( thank you my man and Im gonna close the pull request 

THANKS SO MUCH !

I will make another PR To update readme.... Give me a minute :D lol At least I tried it !

**Pull request closed without merging**